python/src/DirectionalSampling_doc.i.in

```diff
@@ -77,20 +73,25 @@ Examples
 --------
 >>> import openturns as ot
 >>> ot.RandomGenerator.SetSeed(0)
+
+Create the output random vector :math:`Y = \model(\inputRV)`:
 >>> myFunction = ot.SymbolicFunction(['E', 'F', 'L', 'I'], ['-F*L^3/(3*E*I)'])
 >>> myDistribution = ot.Normal([50.0, 1.0, 10.0, 5.0], [1.0]*4, ot.IdentityMatrix(4))
->>> # We create a 'usual' RandomVector from the Distribution
->>> vect = ot.RandomVector(myDistribution)
->>> # We create a composite random vector
->>> output = ot.CompositeRandomVector(myFunction, vect)
->>> # We create an Event from this RandomVector
->>> myEvent = ot.ThresholdEvent(output, ot.Less(), -3.0)
->>> # We create a DirectionalSampling algorithm
+>>> input_RV = RandomVector(myDistribution)
+>>> output_RV = ot.CompositeRandomVector(myFunction, input_RV)
+
+Create the event :math:`\cD_f = \{\inputRV \in \Rset^{\inputDim} \, | \,\model(\vect{x}) \leq -3.0\}`:
+>>> threshold = -3.0
+>>> myEvent = ot.ThresholdEvent(output_RV, ot.Less(), -3.0)
```

Comment:
could you rename these variables ? myEvent -> event

Reply:
yes